pandas/tests/arrays/categorical/test_constructors.py

```diff
@@ -682,3 +682,11 @@ def test_interval(self):
         expected_codes = np.array([0, 1], dtype="int8")
         tm.assert_numpy_array_equal(cat.codes, expected_codes)
         tm.assert_index_equal(cat.categories, idx)
+
+    @pytest.mark.parametrize("nullable", [pd.NA, NaT])
```

Comment:
actually you can use the null_fixture here

Reply:
Thx, done.